bmap-rs/src/main.rs

```diff
@@ -111,6 +111,12 @@ fn find_bmap(img: &Path) -> Option<PathBuf> {
     }
 }
 
+fn find_remote_bmap(url: &Url) -> Result<Url> {
+    let mut bmap = PathBuf::from(url.clone().as_str());
+    bmap.set_extension("bmap");
+    Url::parse(bmap.to_str().unwrap()).map_err(anyhow::Error::new)
```

Comment:
Here I'm parsing from Url to PathBuf and then back to Url. That was already pointed as weird somewhere else. I'm not sure if that's a good way to proceed but the reason to do that is that function `set_extension("bmap")` seems to do exactly what we need, erase the ".gz" extension and add ".bmap". And since we are in the remote side the enum Copy.image here is an Url.